### PR TITLE
Register arceus.is-a.dev

### PR DESCRIPTION
**Pull request by arceusdev**
Register arceus.is-a.dev with CNAME record pointing to blurwind.github.io.

### CODE REVIEW

File: domains/arceus.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "blurwind",
+           "email": "96443442+blurwind@users.noreply.github.com",
```
Comment (CuteDog5695) [inline domains/arceus.json:4]: please replace email line with your own email or leave it blank

File: domains/arceus.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+           "discord": "1055478076013817936"
+        },
+    
+        "record": {
+            "CNAME": "blurwind.github.io"
```
Comment (CuteDog5695) [inline domains/arceus.json:9]: since CNAME record is already redirecting to the is-a.dev domain, can i have a preview of your website?

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by blurwind** [comment]
here is a preview, also available at blurwind.github.io since i removed the CNAME

imageimage

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Comment by blurwind** [comment]
thank you :)

**Pull request merged**